Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 
-ruby ">= 2.4"
+ruby "~> 2.4.0"
```

Comment:
This will break ruby-head no ?

Reply:
Yes. I was wondering why you've changed that. Totally forgot that case. I'll revert it. 😄